### ISSUE TITLE
Commands missing `category`

### ISSUE THREAD

**Issue by usernamehw** [Denis-net/vscode-ahk-manager#11]
https://github.com/Denis-net/vscode-ahk-manager/blob/eaf38971a88b4bb79299bcd3623dd4a5570fff10/package.json#L129-L172

**Comment by usernamehw** [comment]
Not clear where the command is coming from when seeing it the first time from Command Palette of from Keybindings GUI:

![Screenshot (44)](https://user-images.githubusercontent.com/9638156/62972247-45beba80-be1c-11e9-8704-eb7a86b7a016.png)

**Comment by usernamehw** [comment]
Also, if item is not intended to use from `Command Palette` you can exclude it:

```json
"contributes": {
    "menus": {
        "commandPalette": [{
            "command": "extension.sayHello",
            "when": "never"
        }]
    }
}
```
But `category` still helps when using Keybindings GUI.

**Comment by Denis-net** [comment]
Good evening @usernamehw  😊👋🏻

At that time I wondered how extension could hide some command, but didn't find this info.
Now, Thanks to you, the command palette will be a better place ! 😉👍🏻

I spent the holidays solving the problems of the house and renovating the courtyard.
However I made progress with Inkscape, so in the next update coming we will have a better and lighter version (with newer icons).

Thank you, again !
